root/layout.tt

```diff
@@ -91,6 +91,13 @@
         </div>
         [% END %]
 
+        [% IF overlong_string %]
+        <div class="banner warning-header">
+            <p>[% l('Some text you entered is overlong, please fix the text or
+                     shorten it and use annotation for the full text instead.') %]</p>
```

Comment:
"overlong" is not a common word. What about "Some text you've entered is too long for us to support. Please shorten it, and if necessary enter the full text in the annotation for reference." or something like that?

Reply:
> "overlong" is not a common word. What about "Some text you've entered is too long for us to support.

It is not a common word but it isn’t overcomplicated to understand even the first time you read it. The fact that it is not common makes it easier to search the documentation for as we can refer to it in the Editing FAQ.

> Please shorten it, and if necessary enter the full text in the annotation for reference." or something like that?

Committed through https://github.com/metabrainz/musicbrainz-server/pull/3251/commits/ef4fb7e32e76b0ae2651d67becda87062710cf8d.